package.json

```
@@ -51,12 +51,14 @@
     "ferrum": "1.8.0",
     "lodash.pick": "4.4.0",
     "lodash.pickby": "4.6.0",
+    "minimatch": "^3.0.4",
     "xml-escape": "1.1.0",
     "yaml": "1.10.2"
   },
   "devDependencies": {
     "@adobe/eslint-config-helix": "1.1.4",
     "@adobe/helix-cli": "13.3.9",
+    "@adobe/helix-testutils": "^0.4.3",
```

```suggestion
    "@adobe/helix-testutils": "0.4.3",
```

tip: use `npm add -DE @adobe/helix-testutils` to add a pinned dev dependency